src/sentry/tagstore/legacy/backend.py

```diff
@@ -241,7 +241,7 @@ def get_group_tag_values(self, group_ids, environment_id, keys=None, values=None
 
         return list(qs)
 
-    def delete_tag_key(self, project_id, key):
+    def delete_tag_key(self, project_id, environment_id, key):
```

Comment:
Should this pass `environment_id` to `get_tag_key` below even though it's unused?

Reply:
For this impl/backend it doesn't matter, but I cleaned it up anyway.